src/components/tables/Connectors/Rows.tsx

```diff
@@ -54,14 +64,22 @@ function Rows({ data }: Props) {
             {data.map((row) => {
                 return (
                     <TableRow key={`Connector-${row.id}`}>
-                        <TableCell style={columnStyling}>
-                            <ConnectorName connector={row} />
+                        <TableCell align="left" style={columnStyling}>
+                            <ConnectorName
+                                path={row.open_graph['en-US'].image}
+                                connector={row}
+                            />
```

Comment:
Just hardcoding for now. I think we'll need to make a util to grab the proper localized stuff from the open graph. I am thinking we should end up making the open graph stuff fetched on load of the application with SWR and just keep it as a global store.